.github/actions/fetch_submodules/action.yml

```diff
@@ -4,7 +4,7 @@ inputs:
   submodules:
     description: 'The submodules to cache'
     required: false
-    default: '["extmod/ulab", "lib/", "tools/"]'
+    default: '["extmod/ulab", "lib/", "tools/adabot", "tools/bitmap_font", "tools/huffman", "tools/python-semver", "tools/Tecate-bitmap-fonts", "tools/uf2", "tools/usb_descriptor"]'
```

Comment:
Directories are allowed here. Only the submodule paths are fed to `actions/cache`.

Following line does the conversion:
https://github.com/adafruit/circuitpython/blob/72981cf2abf9b58074228062e8d8fdfd1bd53bdc/.github/actions/fetch_submodules/action.yml#L41

From the logs:
```
Run actions/cache@v3
  with:
    path: .git/modules/
  extmod/ulab
  lib/adafruit_floppy
  lib/axtls
  lib/berkeley-db-1.xx
  lib/certificates/nina-fw
  lib/libffi
  lib/mbedtls
  lib/mp3
  lib/nrfutil
  lib/protomatter
  lib/quirc
  lib/tinyusb
  tools/Tecate-bitmap-fonts
  tools/adabot
  tools/bitmap_font
  tools/huffman
  tools/python-semver
  tools/uf2
  tools/usb_descriptor
    key: submodules-common-f01551f27a11d33e88cfb801bd3cbba6613aec14d2b92201488[28](https://github.com/adafruit/circuitpython/actions/runs/4041713302/jobs/6948565517#step:6:30)9f81061fc96
    enableCrossOsArchive: true
```

Reply:
Do you understand why @tannewt found https://github.com/adafruit/circuitpython/commit/78aca074bacb1b428e278adaed9b93240753e68a necessary, then? He saw a build fail that seemed to be due to caching an old version of something (not a submodule) in `tools/`

Reply:
It was an older version of the caching code. Perhaps this new version works ok.

Reply:
aha, ok, I'll close this for now, then